---
layout: documentation
lang-ref: beanbag/68/issue-68-phylogenomics-phylogenetics-working-group
permalink: /beanbag/68/issue-68-phylogenomics-phylogenetics-working-group
title: Phylogenomics & Phylogenetics Working Group
description: Issue 68 - Phylogenomics & Phylogenetics Working Group
sideNavigation: sidenav.beanbag68
---

# Phylogenomics Working Group 

Coordinators:  
**Félix Forest**, Royal Botanic Gardens Kew, UK  
**Erik Koenen**, Université Libre de Bruxelles, Belgium  

***At least eight*** legume phylogenomics projects are underway, as summarized below:  

1. Detarioideae. (M. de la Estralla, D.I. Ojeda, O. Hardy, F. Forest et al.) using a Detarioid-specific bait set for 250 species (one species per genus) plus the Angiosperms353 probes, with a particular focus on the *Berlinia* (D.I. Ojeda, A. Boom, S. Abeele, M. de la Estrella, O. Hardy et al.), *Saraca* (L.M. Choo, M. de la Estrella et al) and *Eperua* (E. Fortes, V. Mansano, J. Doyle, et al.) clades.  
2. Caesalpinioideae. 422 taxa, including 146 of the 151 genera of subfamily Caesalpinioideae have been sequenced using the Mimobaits bait set of 997 genes. The first results for c. 120 taxa have been published (Koenen et al. 2020). The full 422-taxon tree is in preparation for publication. This new backbone phylogeny will be used as the basis for the new tribal / clade-based classification of the subfamily (ALS14 Part 2), for establishing a new generic system for the subfamily (ALS14 Part 1), for investigating phylogenetic turnover and biogeography, and for trait evolution studies. The project is led by Erik Koenen, Jens Ringelberg, Colin Hughes, et al., University of Zürich, Switzerland.  
3. Phylogeny of *Albizia* s.s. and evolutionary origins of savanna species using selected exons from the ***Mimobaits*** probes to produce a new sectional classification, and trait and niche evolution studies in relation to historical assembly of African savannas. Led by E. Koenen, C. Lehmann and O. Hardy.  
4. Phylogenomics and cytonuclear coevolution of Papilionoideae. The aims here are to 1) generate a fully resolved and well supported phylogeny of the major clades of papilionoids using multiple genes from all three genomes, 2) examine coevolution of nuclear, mitochondrial and plastid genes, and 3) determine if changes in nuclear encoded DNA repair, replication and recombination genes are correlated with mitochondrial and plastid genome complexity. Led by R.K. Jansen, T.A. Ruhlman and M.F Wojciechowski, with I.-S. Choi, C. Lee, D. Cardoso, L.P. de Queiroz, and H.C. de Lima.  
5. Phylogenomics of subfamilies Cercidoideae and Dialioideae using Angiosperms353 + 86 genes of known function to resolve phylogenetic relationships, and study trait evolution, biogeography, whole genome duplication events, the fate of duplicated genes, root nodulation gene evolution, correlation between functional genes and trait evolution across biomes, and delineate species complexes. Led by A. Bruneau, W. Cardinal-McTeague, S. Cannon, C. Hagelstam Renshaw, L. Bourgeois-Racette.  
6. Phylogenomics projects from the Ting-Shuang Yi lab, include three main topics: 1) Phylogeny of Leguminosae with 695 accessions including plastomes, mitochondrial and nuclear genes of ~74 tribal-level clades and more than 480 genera to build a solid backbone phylogeny for legumes; 2) plastome evolution: investigating the full history of plastome rearrangements, and the frequency of parallel inversions in different lineages/ clades; 3) whole genome sequencing: plans include seven sequenced species covering all six subfamilies by PacBio/HiFi + HiC + RNAseq and a collaborative plan to sequence the genomes of representatives of all tribes.  
7. Development of a bait set for legumes targeting 507 loci, tested on 25 taxa from across legumes and selected outgroup taxa (Vatanparast et al. 2018). The main aims are target-enrichment phylogenomics to improve the resolution of legume lineages at generic, tribal, and subfamily levels and a generic level phylogenomic backbone of phaseoloid legumes. Led by M. Vatanparast and A.N. Egan.  
8. The legume project under the umbrella of the Plant and Fungal Trees of Life (PAFTOL) project at Kew has so far produced sequence data for 450 genera using the Angiosperms353 probes, most of which will be included in the angiosperm tree of life 2.0 released in December 2021 on the [Kew Tree of Life Explorer](https://treeoflife.kew.org/) (109 genera already available from release 1.0). These genera will also be included in the PAFTOL “big angiosperm tree” paper in preparation. These collaborative efforts involve E. Koenen, F. Forest, G. Lewis, T. Pennington, R. Barrett, A. Bruneau, W. Cardinal-McTeague, D. Cardoso, J. Clugston, A. Egan, M. de la Estrella, B. Klitgaard, T. McLay, D. Murphy, M. Renner, R. Schley, M. Vatanparast, M. Wojciechowski and the PAFTOL team.  


![*Paracalyx scariosus* (Roxb.) Ali, Papilionoideae, photo by Sandip K. Gavade.](/assets/images/68/LPWG-3_Paracalyx.png)
*Paracalyx scariosus* (Roxb.) Ali, Papilionoideae, photo by Sandip K. Gavade.
